WordPress/Classes/Utility/ImmuTableViewController.swift

```diff
@@ -102,12 +88,26 @@ final class ImmuTableViewController: UITableViewController, ImmuTablePresenter {
 
     override func viewWillAppear(animated: Bool) {
         super.viewWillAppear(animated)
-        visibleSubject.on(.Next(true))
+        loadModel()
+        NSNotificationCenter.defaultCenter().addObserver(
+            self,
+            selector: #selector(ImmuTableViewController.loadModel),
+            name: ImmuTableViewController.controllerChangedNotification,
+            object: nil)
+
+        guard !hasAppearedBefore else {
+            return
+        }
+
+        controller.refreshModel()
```

Comment:
Actually I think I'll move it to `init`, since it's not really tied to the view being loaded